src/Bicep.Core/FileSystem/PathHelper.cs

```diff
@@ -23,14 +24,21 @@ public static class PathHelper
         /// </summary>
         /// <param name="path">The path.</param>
         /// <param name="baseDirectory">The base directory to use when resolving relative paths. Set to null to use CWD.</param>
-        public static string ResolvePath(string path, string? baseDirectory = null)
+        /// <param name="fileSystem">The file system abstraction.</param>
+        public static string ResolvePath(string path, string? baseDirectory = null, IFileSystem? fileSystem = null)
```

Comment:
I'm guessing it would have been too big of a refactor to make the filesystem arg mandatory?

Reply:
Indeed, the method is widely referenced. I'm considering a strategy to eliminate `System.IO` and `System.Uri` references from `Bicep.Core`, and the whole `PathHelper` can be removed as part of the change.  I'll draft a REP to outline and discuss the design in detail later.

Reply:
Thanks, makes sense.